MAUI/Segmented-Control/migration.md

```diff
@@ -0,0 +1,603 @@
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+ViewModel viewModel = new ViewModel();
+SfSegmentedControl segmentedControl = new SfSegmentedControl();
+segmentedControl.ItemsSource = viewModel.ItemsCollection;
+segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings() 
+{
+    SelectionIndicatorPlacement = SelectionIndicatorPlacement.Border,
+    Background = Colors.Transparent,
+};
+this.Content = segmentedControl;
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="ViewModel.cs" %}
+
+public class ViewModel
+{
+    private List<SfSegmentItem> itemsCollection;
+
+    public ViewModel()
+    {
+        itemsCollection = new List<SfSegmentItem>()
+        {
+                new SfSegmentItem() {Text="Day"},
+                new SfSegmentItem() {Text="Week"},
+                new SfSegmentItem() {Text="Month"},
+                new SfSegmentItem() {Text="Year"},
+        };
+    }
+
+    public List<SfSegmentItem> ItemsCollection
+    {
+        get { return itemsCollection; }
+        set { itemsCollection = value; }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<table>
+<tr>
+<th>Xamarin SfSegmented control</th>
+<th>.NET MAUI SfSegmented control</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[Color](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SelectionIndicatorSettings.html#Syncfusion_XForms_Buttons_SelectionIndicatorSettings_Color)'| markdownify }}</td>
+<td>Background</td>
+<td>Gets or sets the background brush for the selection indicator in the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>NA</td>
+<td>TextColor</td>
+<td>Gets or sets the text color for the selection indicator in the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>NA</td>
+<td>Stroke</td>
+<td>Gets or sets the stroke color for the selection indicator in the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>{{'[Position](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SelectionIndicatorSettings.html#Syncfusion_XForms_Buttons_SelectionIndicatorSettings_Position)'| markdownify }}</td>
+<td>SelectionIndicatorPlacement</td>
+<td>Gets or sets the selection mode for the selection indicator in the SfSegmentedControl.</td>
+</tr>
+
+<tr>
+<td>{{'[BorderThickness](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html#Syncfusion_XForms_Buttons_SfSegmentedControl_BorderThickness)'| markdownify }}</td>
+<td>StrokeThickness</td>
+<td>Gets or sets the stroke thickness for the selection indicator in the SfSegmentedControl.</td>
+</tr>
+
+</table>
+
+#### SegmentTextStyle
+
+The `FontColor`, `FontSize`, `FontFamily`, `FontAttributes` properties of [SfSegmentedControl](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html) is grouped to `SegmentTextStyle`.
```

Comment:
The `FontColor`, `FontSize`, `FontFamily`, and `FontAttributes` properties of the [SfSegmentedControl](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.SfSegmentedControl.html) is grouped to `SegmentTextStyle`.

Reply:
content changed